lib/peer_manager/modules/peer_database.c

```diff
@@ -38,7 +38,7 @@ LOG_MODULE_DECLARE(peer_manager, CONFIG_PEER_MANAGER_LOG_LEVEL);
 /* Peer Database event handlers in other Peer Manager submodules. */
 extern void pm_pdb_evt_handler(pm_evt_t *p_event);
 extern void sm_pdb_evt_handler(pm_evt_t *p_event);
-#if !defined(CONFIG_PM_SERVICE_CHANGED_ENABLED) || (CONFIG_PM_SERVICE_CHANGED_ENABLED == 1)
+#if !defined(CONFIG_PM_SERVICE_CHANGED) || (CONFIG_PM_SERVICE_CHANGED == 1)
```

Comment:
This does not make sense with the (new) Kconfig system. This logic probably took care of the "default" behavior in the nRF5 SDK, when the config was not defined.
```suggestion
#if defined(CONFIG_PM_SERVICE_CHANGED)
```
Here and others.